src/Tribe/Events/Views/V2/Models/Tickets.php

```diff
@@ -239,7 +239,7 @@ public function offsetExists( $offset ): bool {
 	/**
 	 * {@inheritDoc}
 	 */
-	public function offsetGet( $offset ) {
+	public function offsetGet( $offset ): mixed {
```

Comment:
https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.composite.redundant

You can't use "mixed" here. Use `#[\ReturnTypeWillChange]` in the docblock.